### ISSUE TITLE
Question: Is the MIMC-CXR data related to MIMIC-III clinical database?

### ISSUE THREAD

**Issue by vs74** [MIT-LCP/mimic-cxr#24]
I've a question in regards to the MIMIC-CXR data. I've been wondering is the MIMIC-III clinical database anyway related to the MIMIC-CXR? If it is, then how? 

Actually, I'm interested in vital signs, laboratory reports of the patients of the MIMIC-CXR which are present in the MIMIC-III database, so that's why I asked the question in the first place. 

If there is no relation between the two datasets, then is it possible to get access to BIDMC EHR which was used to collect the x-ray images of MIMIC-CXR data?

Thanks in advance!!

**Comment by alistairewj** [comment]
No, MIMIC-III is entirely separate. MIMIC-IV is the database you are looking for: https://physionet.org/content/mimiciv/

The `PatientID` in the DICOM header of the files is the same as `subject_id` in MIMIC-IV. There is a convenient CSV of MIMIC-CXR metadata in the MIMIC-CXR-JPG project: https://physionet.org/content/mimic-cxr-jpg/

**Comment by vs74** [comment]
Thank you @alistairewj for your insights. I can see that in the MIMIC-CXR-JPG dataset, there are certain patients (**11245** ) who don't have their respective **subject_id** in the MIMIC-IV. For instance, in MIMIC-CXR-JPG dataset patient_id's `12427812`, `14029832` , ..... many more cannot be mapped back to the subject_id's in MIMIC-IV. May I know for these **11245** patients can I get information from MIMIC-IV? If these patients cannot be mapped back to MIMIC-IV data, may I know was the source of data collection for MIMIC-IV and MIMIC-CXR different for these patients?
